### PR TITLE
Update `stern` repo

### PR DESCRIPTION
**Pull request by aknysh**
## what
* Update `stern` repo
* Update GitHub workflows

## why
* Update `stern` repo from https://github.com/wercker/stern (which has been deleted) to https://github.com/stern/stern
* Keep GitHub workflows and READMEs up to date

### CODE REVIEW

**Review by nitrocode — APPROVED** [review]

File: .github/workflows/validate-codeowners.yml [hunk 1]
```diff
@@ -3,18 +3,28 @@ on:
   workflow_dispatch:
 
   pull_request:
-    paths:
-    - '**/CODEOWNERS'
-    - '.github/workflows/validate-codeowners.yml'
```
Comment (Nuru) [inline .github/workflows/validate-codeowners.yml:6]: Revert

File: .github/workflows/validate-codeowners.yml [hunk 2]
```diff
@@ -3,18 +3,28 @@ on:
...
+    - uses: mszostok/codeowners-validator@v0.7.1
+      if: github.event.pull_request.head.repo.full_name == github.repository
+      name: "Full check of CODEOWNERS"
       with:
-        checks: "files,syntax,owners,duppatterns"
+        # For now, remove "files" check to allow CODEOWNERS to specify non-existent
+        # files so we can use the same CODEOWNERS file for Terraform and non-Terraform repos
+        #   checks: "files,syntax,owners,duppatterns"
+        checks: "syntax,owners,duppatterns"
+        owner_checker_allow_unowned_patterns: "false"
```
Comment (Nuru) [inline .github/workflows/validate-codeowners.yml:22]: Revert

File: vendor/stern/VERSION [hunk 3]
```diff
@@ -1 +1 @@
-1.11.0
+1.22.0
```
Comment (Nuru) [inline vendor/stern/VERSION:1]: Do we want to release any intermediate versions?

**Review by Nuru — CHANGES_REQUESTED** [review]

**Review by Nuru — DISMISSED** [review]
Revert automatic changes that were meant for Terraform modules and should not be made here.

**Review by osterman — APPROVED** [review]

**Pull request merged**